### PR TITLE
Deena

### PR DESCRIPTION
**Pull request by Deena2388**
Added 6 unit tests, please review and let me know, should there be any changes.

### CODE REVIEW

File: main.py
Comment (trickaby) [inline main.py:1]: This can be removed

File: tests/test_web_scraper.py [hunk 1]
```diff
@@ -1,21 +1,19 @@
...
     def tearDown(self):
         self.driver.quit()
 
     def test_scrape_page(self):
         title = self.driver.title
-        self.assertEqual(title, "Example Domain")
+        self.assertEqual(title, "Accessibility statement for National Maritime Single Window")
 
     def test_check_header_present(self):
         actual = check_header_present(self.driver, "Example Domain")
```
Comment (trickaby) [inline tests/test_web_scraper.py]: Will need to update this test if we're changing the URL

**Review by trickaby — COMMENTED** [review]
Might be worth removing the .idea folder from vc and adding to .gitignore

File: tests/test_web_scraper.py (hunk 1 above)
Comment (Deena2388) [reply tests/test_web_scraper.py]: Done

**Review by trickaby — APPROVED** [review]

**Pull request merged**